packages/core/admin/admin/src/pages/MarketplacePage/tests/plugins.test.js

```diff
@@ -238,15 +209,13 @@ describe('Marketplace page - plugins tab', () => {
 
   it('filters a category option', async () => {
     const filtersButton = screen.getByTestId('filters-button');
-
     await user.click(filtersButton);
 
     const categoriesButton = screen.getByTestId('Categories-button');
     await user.click(categoriesButton);
 
     const option = screen.getByRole('option', { name: `Custom fields (4)` });
```

Comment:
`const option = screen.getByTestId('Custom fields-4');` this would do the work as test-id is already in place.
<img width="686" alt="Screenshot 2022-11-21 at 14 46 32" src="https://user-images.githubusercontent.com/7886244/203070922-a41fb48e-9bcf-4dd2-9ca1-f80fd37e622f.png">

here I just tried to update 2 lines, but replacing getByRole in this file will save much time.